modules/couchbase/build.gradle

```diff
@@ -0,0 +1,6 @@
+description = "Testcontainers :: Couchbase"
+
+dependencies {
+    compile project(':testcontainers')
+    compile 'com.couchbase.client:java-client:2.5.7'
```

Comment:
`compileOnly` perhaps?

Reply:
Couchbase java client is actually used, so if the client doesn't add this library, it won't run.